lib/user-utilities.coffee

```diff
@@ -109,6 +109,9 @@ module.exports =
     pack = atom.packages.getLoadedPackage(packageName)
     pack?.metadata.version
 
+  getPackageVersionShippedWithAtom: (packageName) ->
+    require(path.join(atom.packages.resourcePath, 'package.json')).packageDependencies[packageName]
```

Comment:
> atom.packages.resourcePath is private, atom.getLoadSettings().resourcePath should be used instead.

Thanks! Fixed in https://github.com/atom/notifications/commit/5cb8e2572c814e619389dc611c6d9f101db0fe27